### DISCUSSION TITLE
Doubt about LEDs

### DISCUSSION THREAD

**Discussion by Prathamkogta** [anatolianroverchallenge/rscp#14] [Q&A] [answered]
During autonomous mission do we need to have two activity lights ? 
One of yellow color(autonomous mission) as mentioned in 3.1.5.1 and other for RSCP sequence as mentioned in the github repository

Team Anveshak

**Comment by senceryazici** [comment, marked as answer]
No, there is only 1 light which can emit green red and yellow colors. The RSCP documentation shows the operation of the LED. If you examine the sequence, the LED emits yellow light during autonomous movement, green at stop, and red when disarmed. Therefore the light should follow the requirements in the manual. There should not be any difference between the requirement in the manual and the RSCP sequence table.
